### DISCUSSION TITLE
i want to download the rabbitmq for arm-architechture ,but i dont konw which assets to choose

### DISCUSSION THREAD

**Discussion by sub-bucket** [rabbitmq/rabbitmq-server#7527] [Other]
Hello,
  i am a develper from china, i installed a centos7 5.11.12-300.el7.aarch64 on VMWare, i want to download the rabbitmq-server on github download page ,but i dont konw which to choose, can you give me some guidance.

  https://github.com/rabbitmq/rabbitmq-server/releases

Thank you.

**Comment by MirahImage** [comment]
Hi, unfortunately, we cannot support CentOS 7, as it does not have a recent enough version of OpenSSL. For more information, see https://www.rabbitmq.com/install-rpm.html#supported-distributions

**Comment by sub-bucket** [comment]
> Hi, unfortunately, we cannot support CentOS 7, as it does not have a recent enough version of OpenSSL. For more information, see https://www.rabbitmq.com/install-rpm.html#supported-distributions

if i swich to centos8 ,which rabbitmq-server assets should i to choose ,i just want download

**Reply by michaelklishin** [reply]
RabbitMQ does not have architecture-specific code. You need to provision a [supported version of Erlang/OTP](https://www.rabbitmq.com/which-erlang.html) for your OS/architecture. I could swear that a RHEL engineer a few months ago had a goal of getting our [zero dependency Erlang RPM](https://github.com/rabbitmq/erlang-rpm/) running on [Fedora Copr](https://developer.fedoraproject.org/) but I don't know where that went and Copr doesn't seem to have package search.

**Reply by sub-bucket** [reply]
So much appreciations!!! Now i got it!!! It's time to sleep in China, cause almost in mid-night, good luck to you,bro!!!

**Comment by MirahImage** [comment]
The instructions are further down the page at https://www.rabbitmq.com/install-rpm.html#downloads
Be sure to verify the downloaded object against the published signature.

**Reply by sub-bucket** [reply]
So much appreciations!!! You're so nice!!!
